.github/workflows/create-release-pr.yml

```diff
@@ -8,7 +8,7 @@ on:
         default: 'main'
         required: true
       release-type:
-        description: 'A SemVer version diff, i.e. major, minor, patch, prerelease etc. Mutually exclusive with "release-version".'
+        description: 'A SemVer version diff, i.e. major, minor, or patch. Mutually exclusive with "release-version".'
```

Comment:
Do we not support prereleases?

Reply:
Not through the `release-type` option. You can still set a specific version through the `release-version` option.